### PR TITLE
Update object IDs to official openrct2 standards

### LINKED ISSUE #56: More descriptive object names
Official OpenRCT2 vehicles are getting long names according to this comment: https://github.com/OpenRCT2/objects/pull/166#issuecomment-1140238893

I think this project is similar in that we are creating names for RCT1 rides. Yes, there are RCT2 versions for most RCT1 rides, but I think the OpenRCT2 long format names should be addressed for this project.

Currently we have a mix of RCT2 names (the ride's name not the full namespace) and new names, following the RCT2 name format.

RCT2 names e.g. rct1aa.ride.bmfl, rct1ll.ride.rftboat

New names e.g. rct1.ride.spcar, rct1.ride.minilady

@gymnasiast what are your opinions on the object ids in this PR?

### LINKED ISSUE #63: Dummy OriginalId
To allow for saving of track designs, dummy OriginalId are required. See this PR for details https://github.com/OpenRCT2/objects/commit/3de5537a848db5770fb97642077dde4fb37f0b42

### PR DESCRIPTION
**Pull request by ZeeMaji**
Updates nearly all of the object IDs to the official ID standards by giving them more descriptive main IDs, along with originalIds for TD6 compatibility.
This also renames all of their object folders accordingly.
Only ones i have left out are the steel mini trains that have already been officially added as i am not sure what we're going to do with vehicles that get added in officially.

This does mean that old saves with the old object ids will no longer load, i plan on making a simple hack build of openrct2 that will allow us to load them and then resave them with the new ids so this hopefully shouldn't be an issue for too long.

I would like @Gymnasiast to review this as well, as this pertains to adding some of these objects to openrct2 officially in the future.

Closes #63 and closes #56

### CODE REVIEW

**Comment by spacek531** [comment]
I think it's best to have the two repos match each other as close as possible, but since they were merged it's not super important, any updates to them can be made in the other repo.

File: ride/rct1.ride.steel_trains/object.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
-    "id": "rct1.ride.steelrc",
-    "version": "1.0",
+    "id": "rct1.ride.steel_trains",
```
Comment (Gymnasiast) [inline ride/rct1.ride.steel_trains/object.json]:
```suggestion
    "id": "rct1.ride.steel_rc_trains",
```

File: ride/rct1.ride.steel_trains_reversed/object.json [hunk 2]
```diff
@@ -1,7 +1,8 @@
 {
-    "id": "rct1.ride.steelrcrv",
-    "version": "1.0",
+    "id": "rct1.ride.steel_trains_reversed",
```
Comment (Gymnasiast) [inline ride/rct1.ride.steel_trains_reversed/object.json]:
```suggestion
    "id": "rct1.ride.steel_rc_trains_reversed",
```

File: ride/rct1.ride.wooden_trains/object.json [hunk 3]
```diff
@@ -1,7 +1,8 @@
 {
-    "id": "rct1.ride.cwood",
+    "id": "rct1.ride.wooden_trains",
```
Comment (Gymnasiast) [inline ride/rct1.ride.wooden_trains/object.json]:
```suggestion
    "id": "rct1.ride.wooden_rc_trains",
```

File: ride/rct1.ride.wooden_trains_reversed/object.json [hunk 4]
```diff
@@ -1,7 +1,8 @@
 {
-    "id": "rct1.ride.cwoodrv",
+    "id": "rct1.ride.wooden_trains_reversed",
```
Comment (Gymnasiast) [inline ride/rct1.ride.wooden_trains_reversed/object.json]:
```suggestion
    "id": "rct1.ride.wooden_rc_trains_reversed",
```

File: ride/rct1aa.ride.steel_mouse_cars/object.json [hunk 5]
```diff
@@ -1,7 +1,8 @@
 {
-    "id": "rct1aa.ride.smouse",
+    "id": "rct1aa.ride.steel_mouse_cars",
```
Comment (Gymnasiast) [inline ride/rct1aa.ride.steel_mouse_cars/object.json]:
```suggestion
    "id": "rct1aa.ride.steel_wild_mouse_cars",
```

**Comment by ZeeMaji** [comment]
> I think it's best to have the two repos match each other as close as possible, but since they were merged it's not super important, any updates to them can be made in the other repo.

As for this i think i'll just leave that as is for now and save the repo parity for another PR.
That being said is there anything else that needs to be done to finish up this PR?

**Comment by spacek531** [comment]
The only thing I would consider is changing `articulated_train` to `wooden_articulated_train`.

**Comment by ZeeMaji** [comment]
Sure thing, i'm also gonna tweak the bumper boat's original id to #RCT1BBT from #RCT1BOT as i think that is a bit better of an originalId for it.

**Comment by spacek531** [comment]
Ok, this looks good. Anything to add @Gymnasiast ?

**Pull request merged**